### PR TITLE
tiltfile: kustomize built-in

### PR DESCRIPTION
**Pull request by jazzdan**

### CODE REVIEW

File: internal/tiltfile/tiltfile_test.go [hunk 1]
```diff
@@ -317,7 +317,7 @@ func TestGetManifestConfigNonFunction(t *testing.T) {
 func TestGetManifestConfigTakesArgs(t *testing.T) {
 	file := tempFile(
 		`def blorgly2(x):
-			return "foo"
+      return "foo"
```
Comment (jazzdan) [inline internal/tiltfile/tiltfile_test.go]: My editor had some weird interaction with indentation, tabs vs spaces and `gofmt`.

**Comment by jazzdan** [comment]
Whoops, gonna add docs to README also.

File: internal/tiltfile/tiltfile.go [hunk 2]
```diff
@@ -248,6 +259,29 @@ func stopBuild(thread *skylark.Thread, fn *skylark.Builtin, args skylark.Tuple,
 	return buildContext, nil
 }
 
+func callKustomize(thread *skylark.Thread, fn *skylark.Builtin, args skylark.Tuple, kwargs []skylark.Tuple) (skylark.Value, error) {
+	var path string
```
Comment (jazzdan) [inline internal/tiltfile/tiltfile.go]: This should probably take a localPath instead of just a string.

Comment (jazzdan) [reply internal/tiltfile/tiltfile.go]: I dunno, what do y'all think? I'm not convinced.

Comment (nicks) [reply internal/tiltfile/tiltfile.go]: can you use the localPathFromSkylarkValue i recently added?

Comment (jazzdan) [reply internal/tiltfile/tiltfile.go]: Yeah I'll do that. I think we should do an audit of the Tiltfile built-ins that we currently have and which ones should take strings and which ones should take localPaths. Maybe only `read_file` should allow a string as a parameter?

**Pull request merged**